internal/daemon/worker/testing.go

```diff
@@ -173,6 +173,9 @@ type TestWorkerOpts struct {
 	// The worker auth KMS to use, or one will be created
 	WorkerAuthKms wrapping.Wrapper
 
+	// The worker credential storage KMS to use, or one will be created
+	WorkerStorage wrapping.Wrapper
```

Comment:
I don't think you've incorporated this into `NewTestWorker(...)` so it's here but I'm not sure it has any affect.  Based on this, perhaps we should add a unit test that makes sure it does have an affect on a TestWorker?

Reply:
I don't see a test for `TestWorker(...)`.  Did I miss something?